src/functionaltest/java/com/ericsson/ei/frontend/TemplateTestCase.java

```diff
@@ -0,0 +1,28 @@
+package com.ericsson.ei.frontend;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+import com.ericsson.ei.frontend.pageobjects.IndexPage;
+import com.ericsson.ei.frontend.pageobjects.TestRulesPage;
+
+public class TemplateTestCase extends SeleniumBaseClass{
+
+    @Test
+    public void testTemplateTestCase() throws Exception {
+
+        //Open indexpage and verify that it is opened
+        IndexPage indexPageObject = new IndexPage(mockedHttpClient, driver, baseUrl);
+        indexPageObject.loadPage();
+        assertEquals("Eiffel Intelligence", indexPageObject.getTitle());
+
+        //The line below contains selenium interaction with mocking
+        String response = "[{\"aggregationtype\":\"eiffel-intelligence\",\"created\":1524037895385,\"notificationMeta\":\"http://eiffel-jenkins1:8080/job/ei-artifact-triggered-job/build\",\"notificationType\":\"REST_POST\",\"restPostBodyMediaType\":\"application/x-www-form-urlencoded\",\"notificationMessageKeyValues\":[{\"formkey\":\"json\",\"formvalue\":\"{parameter: [{ name: 'jsonparams', value : to_string(@) }]}\"}],\"repeat\":false,\"requirements\":[{\"conditions\":[{\"jmespath\":\"gav.groupId=='com.othercompany.library'\"}]}],\"subscriptionName\":\"Subscription1\",\"_id\":{\"$oid\":\"5ad6f907c242af3f1469751d\"}},{\"aggregationtype\":\"eiffel-intelligence\",\"created\":1524037895415,\"notificationMeta\":\"http://eiffel-jenkins1:8080/job/ei-artifact-triggered-job/build\",\"notificationType\":\"REST_POST\",\"restPostBodyMediaType\":\"application/x-www-form-urlencoded\",\"notificationMessageKeyValues\":[{\"formkey\":\"json\",\"formvalue\":\"{parameter: [{ name: 'jsonparams', value : to_string(@) }]}\"}],\"repeat\":false,\"requirements\":[{\"conditions\":[{\"jmespath\":\"gav.groupId=='com.othercompany.library'\"}]}],\"subscriptionName\":\"Subscription2\",\"_id\":{\"$oid\":\"5ad6f907c242af3f1469751e\"}},{\"aggregationtype\":\"eiffel-intelligence\",\"created\":1524223397628,\"notificationMeta\":\"http://<MyHost:port>/api/doit\",\"notificationType\":\"REST_POST\",\"restPostBodyMediaType\":\"application/json\",\"notificationMessageKeyValues\":[{\"formkey\":\"\",\"formvalue\":\"{mydata: [{ fullaggregation : to_string(@) }]}\"}],\"repeat\":false,\"requirements\":[{\"conditions\":[{\"jmespath\":\"submission.sourceChanges[?submitter.group == 'Team Gophers' && svnIdentifier==null]\"}]}],\"subscriptionName\":\"Subscription_Template_Rest_Post_Raw_Body_Json_Trigger\",\"_id\":{\"$oid\":\"5ad9cda5b715d336247e4980\"}}]";
```

Comment:
can it be loaded from file?

Reply:
Absolutely, I didn´t know what way we prefered. There are both positives and negatives about both approaches(i prefer it in the actual tests as it becomes easier to get an overview of what the test is actually doing and returning - the drawback is the cluttering) but if you guys prefer from file i´ll change that.

Reply:
My opinion is to save it in file. Then this data will be more readable and can be reused in other tests, if needed, without coping and pasting this big part of text. But you can also check it with other teammates.

Reply:
Absolutely, i guess its easier with these big responses. I´ll fix that!

Reply:
I prefer this to be in a file also.